src/pages/Team.vue

```diff
@@ -0,0 +1,346 @@
+            >
+              <img :src="node.avatar" alt="" class="h-full w-full" />
+            </div>
+
+            <h2 class="mt-4 font-bold text-xl underline">{{ node.name }}</h2>
+
+            <p class="text-xs text-gray-500 text-center mt-3 ellipsis">
+              {{ node.quote }}
+            </p>
+
+            <ul class="flex flex-row mt-4 space-x-2">
+              <li>
+                <a
+                  v-if="node.email"
+                  target="_blank"
+                  :href="node.email"
+                  class="brand-icon text-green-400"
+                  ><font-awesome :icon="['fas', 'envelope']"
+                /></a>
+              </li>
+              <li>
+                <a
+                  v-if="node.github_handle"
+                  target="_blank"
+                  :href="node.github_handle"
+                  class="brand-icon text-green-400"
+                  ><font-awesome :icon="['fab', 'github']"
+                /></a>
+              </li>
+              <li>
+                <a
+                  v-if="node.linkedin_handle"
+                  target="_blank"
+                  :href="node.linkedin_handle"
+                  class="brand-icon text-green-400"
+                  ><font-awesome :icon="['fab', 'linkedin']"
+                /></a>
+              </li>
+              <li>
+                <a
+                  v-if="node.twitter_handle"
+                  target="_blank"
+                  :href="node.twitter_handle"
+                  class="brand-icon text-green-400"
+                  ><font-awesome :icon="['fab', 'twitter']"
+                /></a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div ref="joinus" class="container relative mx-auto">
+      <div class="items-center flex flex-wrap">
+        <div class="lg:w-6/12 px-4 ml-auto mr-auto text-center">
+          <div class="py-4">
+            <h1 class="font-semibold text-5xl underline">wanna join?</h1>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="p-8 flex items-center justify-center people-card-section">
+      <iframe
+        src="https://docs.google.com/forms/d/e/1FAIpQLScZH-SxBkSNr0ftL4wuuddwsJ5qcdxccMwzmJwiq-uxa-drVA/viewform?embedded=true"
+        width="740"
+        height="724"
+        frameborder="0"
+        marginheight="0"
+        marginwidth="0"
+        >Loading…</iframe
+      >
+    </div>
+  </Layout>
+</template>
+
+<script>
+export default {
+  metaInfo() {
+    return {
+      title: "Team",
+      bodyAttrs: {
+        style: `background-color: ${this.founderData.background_color}; color: ${this.founderData.text_color}`,
+      },
+    }
+  },
+  computed: {
+    memberData() {
+      if (this.$page && this.$page.allTeam && this.$page.allTeam.edges) {
+        return this.$page.allTeam.edges.filter(
+          (elem) => elem.node.designation !== "founder"
+        )
+      } else {
+        return {}
```

Comment:
Should be `[]` no?

Reply:
PR updated as per your feedback.
Please review & resolve this conversation.